crates/bevy_ecs/src/system/commands.rs

```diff
@@ -223,6 +223,29 @@ pub struct Commands {
 }
 
 impl Commands {
+    /// Creates a new entity with a given [DynamicBundle] (which is a 'set' of components) in the current [World].
+    /// 
+    /// # Example
+    /// ```
+    /// // An example with only one component
+    /// commands.spawn(MyComponent);
```

Comment:
you can't spawn single components like this. it must be a tuple. Can we replace it with:

```rust
commands.spawn((MyComponent,));
```
Lets fix this in the other examples too.

Reply:
Ah, damn. I was deceived by 'SpriteComponents' but it is a Bundle. I always wondered why there was a trailing 's'. Now I know. Should be fine now. (hopefully)

Reply:
Yeah you arent the first person. I'm thinking about replacing the "_Components" suffix with "_Bundle" for clarity.